packages/wrangler/src/index.ts

```diff
@@ -290,114 +300,189 @@ export function createCLIParser(argv: string[]) {
 
 	// docs
 	wrangler.command(
-		"docs [command..]",
-		"📚 Open wrangler's docs in your browser",
+		"docs [command]",
+		"📖Open wrangler commands doc in your browser",
 		docsOptions,
 		docsHandler
 	);
 
 	// init
 	wrangler.command(
 		"init [name]",
-		"📥 Initialize a basic Worker project, including a wrangler.toml file",
+		"🔸Innitialize a basic worker application",
 		initOptions,
 		initHandler
 	);
 
-	// generate
-	wrangler.command(
-		"generate [name] [template]",
-		"✨ Generate a new Worker project from an existing Worker template. See https://github.com/cloudflare/templates",
-		generateOptions,
-		generateHandler
-	);
-
-	// [DEPRECATED] build
-	wrangler.command("build", false, buildOptions, buildHandler);
-
-	// [DEPRECATED] config
-	wrangler.command("config", false, noOpOptions, configHandler);
```

Comment:
Removing these outside of a major version is still a breaking change, regardless of whether they were previously deprecated, isn't it?